core/src/main/java/org/apache/spark/memory/TaskMemoryManager.java

```diff
@@ -276,8 +276,6 @@ public long pageSizeBytes() {
    *
    * Returns `null` if there was not enough memory to allocate the page. May return a page that
    * contains fewer bytes than requested, so callers should verify the size of returned pages.
-   *
-   * @throws TooLargePageException
```

Comment:
Another caveat I didn't mention is that it _is_ a little useful to document _unchecked_ exceptions this way, as only checked exceptions appear in the javadoc. I'd leave this, even without a description.

Reply:
oh, my mistake.